webpage/index.rst

```diff
@@ -31,6 +31,8 @@ This site is organised in the following sections:
    :caption: User Documentation
 
    use/index
+   use/source_list
+   use/publication_list
```

Comment:
The subsections of user documentation have the same structure as in contributors documentation. One page per subsection and going the previous/next subsection by clicking on the arrows at the end of the page.
Think it is better to have one structure in the whole webpage.

Reply:
Please keep the separation in user and contributor documentation. The contributor documentation contains details about the formats, the Python scripts, Github, .... Users shouldn't have to worry about that.